HumanDaoFactory.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.factory;
+
+import mate.academy.dao.Dao;
+import mate.academy.dao.FileHumanDao;
+import mate.academy.dao.InMemoryHumanDao;
+import mate.academy.service.PropertyLoader;
+
+import java.io.IOException;
+
+public class HumanDaoFactory {
+
+    private static final Dao inMemoryDao = new InMemoryHumanDao();
+    private static final Dao fileDao = new FileHumanDao();
+
+    public static Dao getHumanDao(boolean isFileDao, boolean isInMemoryDao) {

```

Comment:
what would happen if call getHumanDao(true, true) ?

Reply:
> what would happen if call getHumanDao(true, true) ?

the choice of record depends on what is written in the file (application.properties)

Reply:
in such case why this prams for ?

Reply:
> in such case why this prams for ?
To check whether you can use one or another way of recording through the annotations